teamsyncd/teamsync.cpp

```diff
@@ -119,10 +203,17 @@ void TeamSync::onMsg(int nlmsg_type, struct nl_object *obj)
         return;
     }
 
+    /* 
+     * We are here because this is a NEW link create netlink message from kernel. 
+     * Fetch and compare the interface name using ifindex from kernel cache. 
+     */
+    if(!checkIfindexToName(ifindex, lagName)) {
```

Comment:
under what conditions do we see ifindex does not match lagName? if we ignore such messages, what is the consequence here? Are we losing some validate netlink messages?

Reply:
From my testing I have seen messages with the older if indices coming to teamsyncd on doing a config reload. So the idea was to make sure we proceed to add Lag only the ifindices --> interface name and the "Name" which came in netlink message is same. This check should not resulting in losing valid messages.